test/Mono.Linker.Tests.Cases/Reflection/TypeUsedViaReflection.cs

```diff
@@ -359,5 +361,23 @@ static Type GetTypeFromAssembly (Assembly assembly, string name, bool caseSensit
 		{
 			return assembly == null ? Type.GetType (name, caseSensitive) : assembly.GetType (name, caseSensitive);
 		}
+
+		[Kept]
+		[ExpectedWarning ("IL2096", "'System.Type.GetType(String,Boolean,Boolean)'")]
+		static void TestUnkownIgnoreCase3Params (int num)
```

Comment:
Do we have coverage for similar cases for other intrinsics?

Reply:
I checked all other cases where we consume the "method parameter" value in the intrinsic handling and none has this problem. We probably don't have coverage though. There are lot of places where we consume binding flags which could all be "null" - and the code has handling for it.

I'm adding a test for the null case for binding flags in one place - but all the other places use the same helper to handle this, so in terms of code coverage it's good. Writing one for each case would be lot of new test cases.